edx_proctoring/tests/test_api.py

```diff
@@ -1447,30 +1425,32 @@ def test_update_attempt_without_credit_state(self):
                 'in_completed_state': True
             }
         ),
-        (
-            ProctoredExamStudentAttemptStatus.created, {
-                'status': ProctoredExamStudentAttemptStatus.created,
-                'short_description': 'Taking As Proctored Exam',
-                'suggested_icon': 'fa-pencil-square-o',
-                'in_completed_state': False
-            }
-        ),
-        (
-            ProctoredExamStudentAttemptStatus.download_software_clicked, {
-                'status': ProctoredExamStudentAttemptStatus.download_software_clicked,
-                'short_description': 'Taking As Proctored Exam',
-                'suggested_icon': 'fa-pencil-square-o',
-                'in_completed_state': False
-            }
-        ),
-        (
-            ProctoredExamStudentAttemptStatus.ready_to_start, {
-                'status': ProctoredExamStudentAttemptStatus.ready_to_start,
-                'short_description': 'Taking As Proctored Exam',
-                'suggested_icon': 'fa-pencil-square-o',
-                'in_completed_state': False
-            }
-        ),
+        # if you call `update_exam_attempt` with the following statues, it will submit the
+        # exam automatically. We need to update these tests.
```

Comment:
Theses tests need an update. I am not sure about the `get_attempt_status_summary` usage in the application. I am open to discussion